README/ReleaseNotes/v616/index.md

```diff
@@ -158,9 +157,12 @@ See `TFile::GetStreamerInfoListImpl` implementation for an example on how to imp
   - Create names for histograms and graphs based on the input columns if no model is provided.
   - RCutFlowReport can print cumulative efficiency of cuts.
   - Reading and writing of columns holding `vector<bool>` instances and `bool` C arrays.
-  - Support `rdfentry_` and `rdfslot_` implicit columns.
+  - Support `rdfentry_` and `rdfslot_` implicit columns (`tdfentry_` and `tdfslot_` kept for backwards compatibility).
   - Remove `RDataFrame` from the 32-bit builds.
   - Speed up interpreted usage of RDataFrame (i.e. in macros or from ROOT prompt) by removing certain cling runtime safety checks.
+  - Streamline and better document usage of multi-thread RDataFrame: edge cases in which processing of an event could start
+    before processing of another event finished have been removed, making it easier for user to write safe parallel RDF operations. 
+    See the [relevant documentaion](https://root.cern.ch/doc/master/classROOT_1_1RDataFrame.html#parallel-execution) for more information.
```

Comment:
Should read "relevant **documentation**"

Reply:
thanks, fixed!